csrc/scheduler/normalization_inner_outer.cpp

```diff
@@ -1103,9 +1109,10 @@ void scheduleReductionCombinedOuter(
             axisID, NamedScalar::getParallelDim(ParallelType::TIDx));
         outer_reduction_tv->axis(axisID--)->parallelize(ParallelType::TIDx);
       }
-
-      outer_reduction_tv->split(
-          axisID, NamedScalar::getParallelDim(ParallelType::BIDy));
+      if (rparams.combined_split_grid_inner_dim) {
```

Comment:
naive question. Even though we are skipping the split, I thought we still would need to specify the current IterDomain with `ParallelType::BIDy`?

Reply:
Yes, we always specify it as `ParallelType::BIDy` using `outer_reduction_tv->axis(axisID--)->parallelize(ParallelType::BIDy);`

Reply:
`outer_reduction_tv->axis(axisID--)->parallelize(ParallelType::BIDy);` is out of the `if` statement.